### PR TITLE
OKTA-515111 Added steps to Add a Groups claim for the Org Authorization Server se…

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Added steps to "Add Groups claim for the Org Authorization Server" section of [Customize tokens returned from Okta with a Groups claim](https://developer.okta.com/docs/guides/customize-tokens-groups-claim/-/main/#add-a-groups-claim-for-the-org-authorization-server) guide.
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-515111](https://oktainc.atlassian.net/browse/OKTA-515111)

### CODE REVIEW

**Review by joelfranusic-okta — APPROVED** [review]
LGTM :shipit:

**Review by joelfranusic-okta — APPROVED** [review]
Approved

**Review by susanharper-okta — APPROVED** [review]
1 nit - approving to not hold you up.

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-groups-claim/main/index.md [hunk 1]
```diff
@@ -113,6 +113,8 @@ Use these steps to create a Groups claim for an OpenID Connect client applicatio
 1. In the **Groups claim type** section, you can select either **Filter** or **Expression**. For this example, leave **Filter** selected.
 1. In the **Group claims filter** section, leave the default name `groups` (or add it if the box is empty), and then add the appropriate filter. For this example, select **Matches regex** and enter `.*` to return all of the user's Groups. See [Okta Expression Language Group Functions](/docs/reference/okta-expression-language/#group-functions) for more information on expressions.
 1. Click **Save**.
+1. Click the **Back to applications** link.
+1. From the **More** button dropdown menu, click **Refresh Application Data**.
```
[the remark this answers is not in the document]
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/customize-tokens-groups-claim/main/index.md:117]: Ignoring as per our Slack discussion. :o)

**Pull request merged**